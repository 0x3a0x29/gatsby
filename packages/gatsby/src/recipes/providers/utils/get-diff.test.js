const getDiff = require(`./get-diff`)

const oldString = `
pizza
super
duper
`

const newString = `
pizza

duper
`

it(`diffs strings by line with color codes`, async () => {
  const result = await getDiff(oldString, newString)
  expect(result).toMatchSnapshot()
})
