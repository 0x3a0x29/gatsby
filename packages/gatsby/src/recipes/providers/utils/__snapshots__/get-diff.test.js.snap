// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`diffs strings by line with color codes 1`] = `
"[31m- Original  - 1[39m
[32m+ Modified  + 1[39m


[2m  pizza[22m
[31m- super[39m
[32m+[39m
[2m  duper[22m
"
`;
